contributing_to_docs/templates/topic_template.adoc

```diff
@@ -12,4 +12,4 @@ toc::[]
 == Overview
 --All topics should include an Overview section that introduces the topic. Remove this and add a brief introduction here.--
 
-== Section Title
+== Section Title
```

Comment:
Probably removing the newline from eof was a typo?
It seems asciidoctor doesn't care, but for future-proofing against lame tools, we probably want to maintain eof newline.
Reverting this hunk also makes for a smaller diff, always a good thing.  :)